tests/conftest.py

```diff
@@ -125,7 +124,7 @@ def android_emu_driver(request, data_center):
         'deviceName': 'Android GoogleAPI Emulator',
         'platformVersion': '10.0',
         'platformName': 'Android',
-        'app': "https://github.com/sentry-demos/sentry_react_native/releases/download/1.8/app-release.apk",
+        'app': "https://github.com/sentry-demos/sentry_react_native/releases/download/1.9/app-release.apk",
         'sauce:options': {
```

Comment:
I'd like to update this so that we fetch a list of releases for the `sentry_react_native` repo via the Github API, and select the most recent one -- so that we don't have to tick this value manually anymore. 

However, don't want to slow this down so I can work on that separately.

Reply:
ok sounds good